config/smartnoise-synth.sh

```diff
@@ -0,0 +1 @@
+PYTHON_DEPS="poetry"
```

Comment:
This config should not be needed as `poetry` is provided by `python-build-bundle`.

Reply:
`MODULE_BUILD_DEPS="python-build-bundle"` therefore ?

Reply:
You can remove entirely the config.